### ISSUE TITLE
imap.yandex.com

### ISSUE THREAD

**Issue by beeflin** [AdguardTeam/AdguardFilters#24610]
[//]: # (***You can delete or ignore strings starting with "[//]:" They will not be visible either way.)

***Description***:
* **Current behaviour**: 
Gmail can't sign into imap.yandex.com when Adguard is enabled. Not sure whether, or what, to whitelist? Gmail or imap.yandex.com? 

[//]: # (Replace %screenshot_url% below with a link to the screenshot of the problem. Also, you can paste image from clipboard instead. It will be automatically loaded.)
<details><summary>Screenshot:</summary>

![image](https://photos.app.goo.gl/SSLGxETradwedtZVA)
</details><br/>

* **Expected behaviour**: 

[//]: # (Substitute this line with a description of what should happen normally. If needed, provide a screenshot below, same as above)

<details><summary>Screenshot:</summary>

![image](%url_of_screenshot%)
</details><br/>

***Steps to reproduce the problem***:

[//]: # (Substitute this line with a step-by-step instruction how to reproduce the issue)

***System configuration***

**Filters:**

[//]: # (Substitute this line with the list of your active filters, separated by commas)

[//]: # (Please enter the correct values for your case to the table below)

Information                            | Value
---                                    | ---
Operating system:                      | ?
Operating system version (Android/iOS) | Oreo 8.0
Browser:                               | ?
AdGuard version:                       | ?
Filters enabled:                       | ?
AdGuard mode (Android only):           | VPN / Proxy (manual/auto)
Filtering quality (Android only):      | High-quality / High-speed / Simplified
HTTPS filtering (Android only):        | On (Default/Blacklist mode) / Off
Simplified filters (iOS only)          | On / Off
AdGuard DNS:                           | None / Default / Family Protection
Stealth mode options (Windows only)    | ?
Helpdesk ID (if exists):               | ?

[//]: # (This template is meant for missed ad/false positive reports, for other type of reports edit it accordingly)
[//]: # (If this is a crash report, include the crashlog with https://gist.github.com/)

**Comment by Alex-302** [comment]
Hi
I need more detailed description of your actions.

**Comment by beeflin** [comment]
Sorry, I failed to upload my screenshot.

I was trying to add my Yandex email account to the Gmail app. 

I entered my username and password, the incoming server address imap.yandex.com, port 993, and the correct security protocol, which is SSL/TLS. When I tried to save, I got a "couldn't open connection to server" error.

When I paused Adguard and tried again, it worked.

However, I haven't needed to leave Adguard off to receive email. It only needed to be paused when adding the account.
